test/pow/plug_test.exs

```diff
@@ -84,21 +85,6 @@ defmodule Pow.PlugTest do
     end
   end
 
-  test "delete/1" do
```

Comment:
Unrelated, but this was unnecessary since https://github.com/danschultzer/pow/pull/405.